### PR TITLE
update caveats + raw data

### PR DESCRIPTION
**Pull request by rickyyx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,8 +30,24 @@ For each provider, we perform:
...
+
+- The endpoints provider backend might vary widely, so this is not a reflection on how the software runs on a particular hardware.
+- The results may vary with time of day.
+- The results (e.g. measurement of TTFT) depend on client location, and can also be biased by some providers lagging on the first token in order to increase ITL. Our current measurement location is from us-west (Oregon).
+- The results is only a proxy of the system capabilities and is also impacted by the existing system load and provider traffic.
+- The results may not correlate with users’ workloads.
+
+
+
+>>>>>>> Stashed changes
```
Comment (xieus) [inline README.md]: Is there some merge conflict here? 

see line 39 and line 50.

File: README.md [hunk 2]
```diff
@@ -30,8 +30,24 @@ For each provider, we perform:
...
+
+<<<<<<< Updated upstream
+=======
+Note that there may be some possible source of biases or discrepancies from your perceived behavior:
+
+- The endpoints provider backend might vary widely, so this is not a reflection on how the software runs on a particular hardware.
+- The results may vary with time of day.
+- The results (e.g. measurement of TTFT) depend on client location, and can also be biased by some providers lagging on the first token in order to increase ITL. Our current measurement location is from us-west (Oregon).
+- The results is only a proxy of the system capabilities and is also impacted by the existing system load and provider traffic.
+- The results may not correlate with users’ workloads.
```
Comment (xieus) [inline README.md:44]: > The results may not correlate with users’ workloads.


Should we mention ShareGPT, the public data set used to benchmark the different provider?

Comment (rickyyx) [reply README.md:44]: We can - it's part of LLMPerf's, so it's documented there. But we could callout here if people have been asking about what datasets we used.

File: README.md (hunk 1 above)
Comment (rickyyx) [reply README.md]: yep - i forgot to push. thanks.

**Pull request closed without merging**